react/Modal/index.jsx

```diff
@@ -8,6 +8,11 @@ const ModalContent = ({children, className}) =>
     {children}
   </div>)
 
+const ModalContentOffset = ({children, className}) =>
```

Comment:
I'm very confused about the different types of modal contents, which ones have padding, which ones don't, and when to use what. Nothing wrong with the code, just a comment.